fmf/base.py

```diff
@@ -220,6 +221,9 @@ def _merge_minus(self, data, key, value):
     def _merge_special(self, data, source):
         """ Merge source dict into data, handle special suffixes """
         for key, value in sorted(source.items()):
+            # Merge only attribute keys, not the adjust control keys
+            if key in ADJUST_CONTROL_KEYS:
+                continue
```

Comment:
The `_merge_special()` method is used in `Tree.merge()` as well where child data are merged with parent data so this would break regular metadata using one of the reserved keys. For example:

```yaml
what: something

/child:
    when: yesterday
```

Before the change we get:

    > fmf show
    /child
    what: something
    when: yesterday

But after the child value is not included.

    > fmf show
    /child
    what: something

Reply:
ah. thanks for catching this.

Reply:
Or, what about just filtering the extra keys when calling the method in the `adjust` case?

```python
self._merge_special(
    self.data, {
        key: value
        for key, value in rule.items()
        if key not in ADJUST_CONTROL_KEYS
        }
    )
```

Reply:
fixed in 9cef668

Reply:
Good, thanks!